app/assets/javascripts/discourse/app/components/quote-button.js

```diff
@@ -199,6 +200,12 @@ export default Component.extend(KeyEnterEscape, {
 
     if (showAtEnd) {
       this.popperPlacement = "bottom-start";
+
+      if (isAndroid) {
```

Comment:
Did you test with iOS? It may be helpful to add the larger offset there as well, I remember in the past that we had similar issues in iOS.

Reply:
Yes, tested on iOS as well. There is an offset there as well, it's just the Android one is a lot bigger so it has a larger offset which doesn't look so good on iOS as its not needed.

See screenshots in OP, but in short we have:

`[0, 3]` offset for desktop (default)
`[0, 15]` offset for mobile/iOS
`[0, 25]` offset for Android

Reply:
Cool, it might be OK to simplify this to 2 states, and use 25px offset for all mobile use cases.